content/posts/blog/first-krakow-xmpp-meetup.md

```diff
@@ -0,0 +1,20 @@
+---
+title: First Krakow XMPP meetup
+date: 2017-04-28 12:59
+author: Nÿco
+blog_id: blog
+category: Events
+---
+
+Following and reinforcing the trend of XMPP meetups across Europe (Berlin, Stockholm, London, and Paris), here is Kraków!
+
+Tigase and MongooseIM teams will join forces to offer you three presentations:
+* "Why use XMPP for IoT?" by Andrzej Wojcik from Tigase
```

Comment:
An empty line is needed before (and after, I think) a list of bullet points, otherwise the list won't render as a list.